SPECS/nbdkit/nbdkit.spec

```
@@ -0,0 +1,2116 @@
+Install this package if you want intelligent bash tab-completion
+for %{name}.
+
+%prep
+%autosetup -p1
+%if 0%{patches_touch_autotools}
+autoreconf -i
+%endif
+
+
+%build
+# Golang bindings are not enabled in the build since they don't
+# need to be.  Most people would use them by copying the upstream
+# package into their vendor/ directory.
+export PYTHON=python3
+%configure \
+    --with-extra='%{name}-%{version}-%{release}' \
+    --disable-static \
+    --disable-golang \
+    --disable-rust \
+    --disable-ruby \
+    --disable-ssh \
+%if 0%{?have_ocaml}
+    --enable-ocaml \
+%else
+    --disable-ocaml \
+%endif
+%if 0%{?have_libguestfs}
+    --with-libguestfs \
+%else
+    --without-libguestfs \
+%endif
+%ifarch %{complete_test_arches}
+    --enable-libguestfs-tests \
+%else
+    --disable-libguestfs-tests \
+%endif
+    --with-tls-priority=@NBDKIT,SYSTEM
+
+#    --disable-lua \
+#    --disable-perl \
+#    --disable-tcl \
+#    --without-ext2 \
+#    --without-iso \
+#    --without-libvirt
+
+# Verify that it picked the correct version of Python
+# to avoid RHBZ#1404631 happening again silently.
+grep '^PYTHON_VERSION = 3' Makefile
+
+%make_build
+
+# Delete the S3 plugin as it is not built
+rm -f %{buildroot}%{_mandir}/man1/nbdkit-S3-plugin*
+rm -f %{buildroot}%{_libdir}/%{name}/plugins/nbdkit-S3-plugin*
+
+
+%install
+%make_install
+
+# Delete libtool crap.
+find %{buildroot} -type f -name "*.la" -delete -print
+
+# If cargo happens to be installed on the machine then the
+# rust plugin is built.  Delete it if this happens.
+rm -f %{buildroot}%{_mandir}/man3/nbdkit-rust-plugin.3*
+
+# Delete the S3 plugin as it is not built
+rm -f %{buildroot}%{_mandir}/man1/nbdkit-S3-plugin*
+rm -f %{buildroot}%{_libdir}/%{name}/plugins/nbdkit-S3-plugin*
+
+
+%check
+%ifnarch %{broken_test_arches}
+function skip_test ()
+{
+    for f in "$@"; do
+        rm -f "$f"
+        echo 'exit 77' > "$f"
+        chmod +x "$f"
+    done
+}
+
+# Workaround for broken libvirt (RHBZ#1138604).
+mkdir -p $HOME/.cache/libvirt
+
+# tests/test-captive.sh is racy especially on s390x.  We need to
+# rethink this test upstream.
+skip_test tests/test-captive.sh
+
+%ifarch s390x
+# Temporarily kill tests/test-cache-max-size.sh since it fails
+# sometimes on s390x for unclear reasons.
+skip_test tests/test-cache-max-size.sh
+%endif
+
+# Temporarily kill test-nbd-tls.sh and test-nbd-tls-psk.sh
+# https://www.redhat.com/archives/libguestfs/2020-March/msg00191.html
+skip_test tests/test-nbd-tls.sh tests/test-nbd-tls-psk.sh
+
+# This test fails on RHEL 9 aarch64 & ppc64le with the error:
+# nbdkit: error: allocator=malloc: mlock: Cannot allocate memory
+# It could be the mlock limit on the builder is too low.
+# https://bugzilla.redhat.com/show_bug.cgi?id=2044432
+#%ifarch aarch64 %{power64}
+#skip_test tests/test-memory-allocator-malloc-mlock.sh
+#%endif
+
+# Make sure we can see the debug messages (RHBZ#1230160).
+export LIBGUESTFS_DEBUG=1
+export LIBGUESTFS_TRACE=1
+%endif
```

I think we're missing the part, that actually runs the tests. From Fedora 36's specs I see we could use this bit:
```suggestion

%make_build check || {
    cat tests/test-suite.log
    exit 1
  }
%endif
```

---

Done!